packaging/rpm/mackerel-agent-systemd.spec

```diff
@@ -6,14 +6,14 @@
 Name:      mackerel-agent
 Version:   %{_version}
 Release:   1%{?dist}
-License:   Commercial
+License:   ASL 2.0
```

Comment:
https://fedoraproject.org/wiki/Packaging:LicensingGuidelines

> The License: field must be filled with the appropriate license Short License identifier(s) from the "Good License" tables on the Fedora Licensing page.

And Short License Identifier for Apache License 2.0 is `ASL 2.0`, according to the table.
https://fedoraproject.org/wiki/Licensing:Main